### ISSUE TITLE
Usage of align_faces.py

### ISSUE THREAD

**Issue by egorpezdir** [yangxy/GPEN#133]
It is preferred that the images are well aligned for the colorization. But how do i do it with `align_faces.py`?

**Comment by yangxy** [comment]
Yes. You should first detect faces along with their 5-point landmarks using retinaface detector. And then for each detected face, use the function `warp_and_crop_face` in `align_faces.py` to obtain its corresponding aligned face. Please check the codes in `face_enhancement.py`.
